internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -5063,6 +5063,49 @@ func TestHandleGetTxOutSetInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleInvalidateBlock(t *testing.T) {
+	t.Parallel()
+
+	chainWithErr := func(err error) *testRPCChain {
+		chain := defaultMockRPCChain()
+		chain.invalidateBlockErr = err
+		return chain
+	}
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleInvalidateBlock: ok",
+		handler: handleInvalidateBlock,
+		cmd:     &types.InvalidateBlockCmd{},
```

Comment:
Do you think there should be a valid hash included in the command here?  I see that the test is still following the expected path since `chainhash.NewHashFromStr` returns a zero hash for an empty string, but it might be good to include a hash so it more closely resembles an actual command that would be run:
```suggestion
		cmd: &types.InvalidateBlockCmd{
		    BlockHash: block432100.BlockHash().String(),
		},
```

Reply:
Done.